chapter1/README.md

```diff
@@ -1,3 +1,22 @@
 # Walk to the Chapter
 
 This Chapter will talk about every bit and byte of ManageIQ. It will serve as guidelines for all those folks who wish to start with it. The Chapter starts with the concept of ManageIQ and ends with its full installation. It's an kind request to all the readers that they make sure the mentioned recommendations of infrastructure are properly met unless otherwise the system may hang completely.
+
+Let's quickly go through the stuffs discussed within the chapter.
+
+##### 1. Introduction to ManageIQ
+
+This part is all about the concept of ManageIQ. It makes the reader fully aware about it's ability and capabilities.
+
+##### 2. Installation
+
+Here installation of ManageIQ used during the project is discussed thoroughly.
```

Comment:
Can you please elaborate this because this actually does get.

Reply:
May I Know what exactly do you wish, because I don't think this is the right place to elaborate ?

Reply:
I also think this is not right place to elaborate. Because it is useless to describe installation at time of introduction of chapter and given description is sufficient for initial knowledge.

Reply:
I mean, please change the sentence formation is bit straight way. Like "Installation of ManageIQ is discussed here, that is used in the project."

Reply:
She wrote it better way in my opinion.

Reply:
Even I think the sentence is okay, see if the suggested change is really required.

Reply:
Ooh.. Great going then. No Comments.

Reply:
Great going the.